server/app/views/components/FieldWithLabel.java

```diff
@@ -246,100 +262,158 @@ public FieldWithLabel showFieldErrors(boolean showFieldErrors) {
     return this;
   }
 
-  public ContainerTag getContainer() {
+  protected void genRandIdIfEmpty() {
```

Comment:
Maybe reorder methods so that diff shows that `getContainer` was not deleted or made `protected`/`private`